src/notebook_unit1.ipynb

```diff
@@ -95,7 +95,7 @@
    "source": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    d = pd.DataFrame(
</pre><p>"d" is not a very explicit variable name I would prefer something like "euclid_dist" for readability</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pauleonix/ssd_course/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>